### PR TITLE
Add SPC mainnet eip155-575757.json

### PR DESCRIPTION
**Pull request by haidarabdillah**
add SPECTA (SPC) mainnet

### CODE REVIEW

**Comment by haidarabdillah** [comment]
seem i forgoty the coma, change it sir

File: _data/chains/eip155-575757.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Spectachain",
+  "chain": "SPECTA",
+  "rpc": [
+    "https://rpc.spectachain.io/"
+  ],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Spectachain",
```
Comment (MRSKELVIN) [inline _data/chains/eip155-575757.json:9]: Withdraw

**Review by MRSKELVIN — APPROVED** [review]

**Review by Jess25051980 — APPROVED** [review]

**Review by Jess25051980 — APPROVED** [review]

**Review by Jess25051980 — APPROVED** [review]

**Comment by haidarabdillah** [comment]
always error due not updated sir?

**Review by Jess25051980 — APPROVED** [review]

**Comment by haidarabdillah** [comment]
why getting update again sir?

**Comment by IGhottem** [comment]
Hello

**Review by foniaharris12 — COMMENTED** [review]
Trying to learn how

**Comment by haidarabdillah** [comment]
anyone can help me to add this?

**Review by foniaharris12 — COMMENTED** [review]
I like it

**Comment by haidarabdillah** [comment]
anyone nca help?

**Pull request closed without merging**